### DISCUSSION TITLE
如何改变子应用返回的页面

### DISCUSSION THREAD

**Discussion by westwind027** [jd-opensource/micro-app#302] [Q&A]
是否有方法能修改子应用返回的页面，在beforemount里面的通过document处理，出现问题

![image](https://user-images.githubusercontent.com/9116046/156392648-8255062b-caad-4b42-8839-9b065da04b2e.png)

![image](https://user-images.githubusercontent.com/9116046/156392748-751d0715-5e11-47c9-b003-8e3d304890da.png)

想删除第一个htmlcollection,会报错。

**Comment by bailicangdu** [comment]
只能通过路由控制
